src/builder.rs

```diff
@@ -116,8 +116,7 @@ impl<'a, 'gcc, 'tcx> Builder<'a, 'gcc, 'tcx> {
 
         // NOTE: since jumps were added and compare_exchange doesn't expect this, the current blocks in the
```

Comment:
Please update `blocks` to `block` in this comment and the others as there is only one block now.